doc/HLD_of_SNS_Repair.rst

```diff
@@ -0,0 +1,1622 @@
+High Level Design of SNS Repair
+===============================
+
+
+This document provides a High-Level Design (HLD) of SNS repair for Motr M0. The main purposes of this document are:
+
+- To be inspected by M0 architects and peer designers to make sure that HLD is aligned with M0 architecture and other designs, and contains no defects
+- To be a source of material for Active Reviews of Intermediate Design (ARID) and Detailed Level Design (DLD) of the same component
+- To serve as a design reference document
+
+The intended audience of this document consists of Motr customers, architects, designers and developers.
+
+
+0. Introduction
+---------------
+
+.. **Note:** This document has been revised to exclude the copy machine related information. The copy machine design has been included in a separate document HLD of copy machine and agents.
+
+Redundant striping is a proven technology to achieve higher throughput and data availability. The Server Network Striping (SNS) applies this technology to network devices and achieves similar goals as a local RAID.
+In the case of storage and/or server failure, the SNS repair reconstructs the lost data from survived data reliably and quickly, without major impact to the production systems. This document presents the HLD of SNS repair.
+
+
+1. Definitions
+--------------
+
+.. Definitions of terms and concepts used by the design go here. The definitions must be as precise as possible. References to the `Motr Glossary<Link-to-be-added>`__ are permitted and encouraged. Agreed upon terminology should be incorporated in the glossary.
```

Comment:
`Motr Glossary </doc/PDF/Glossary.pdf >`__

Reply:
Actually, I made these changes already, but I think you over-wrote it in your commits.

Reply:
Updated.